webui/PCWConfig.py

```diff
@@ -83,7 +85,7 @@ def get_feature_property(feature: str, property: str, namespace: str = None):
     def get_namespaces_for(feature: str) -> list:
         if PCWConfig.has(feature):
             return ConfigFile().getList(f'{feature}/namespaces', ConfigFile().getList('default/namespaces'))
-        return list()
+        return []
```

Comment:
I remember that I was corrected by some linter to actually use `list()` :thinking:

Reply:
you just resolved w/o putting answer :)

Reply:
> you just resolved w/o putting answer :)

I think in this case we have to agree with the linter, even if it contradicts a former version of itself. :)

Reply:
ah ok I get the point now . yeah "Rule 1: Linter always right Rule 2: when linter not right see Rule 1" :D